torch/_torch_docs.py

```diff
@@ -104,16 +107,16 @@ def merge_dicts(*dicts):
 
 .. math::
     \text{out}_{i} = |\text{input}_{i}|
-
+""" + r"""
 Args:
-    input (Tensor): the input tensor
-    out (Tensor, optional): the output tensor
+    {input}
+    {out}
 
 Example::
 
     >>> torch.abs(torch.tensor([-1, -2, 3]))
     tensor([ 1,  2,  3])
-""")
+""".format(**factory_common_args))
```

Comment:
I think the change is good. It's better if you can create a more general arg template to encompass these little '{input}' and '{output}' instead of relying on the semantically oblique `factory_common_args`.